deployment/ccip/changeset/cs_chain_contracts.go

```diff
@@ -4,35 +4,175 @@ import (
 	"bytes"
 	"context"
 	"encoding/hex"
+	"errors"
 	"fmt"
 	"math/big"
 
 	"github.com/ethereum/go-ethereum/accounts/abi/bind"
 	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/smartcontractkit/ccip-owner-contracts/pkg/gethwrappers"
 	"github.com/smartcontractkit/ccip-owner-contracts/pkg/proposal/mcms"
 	"github.com/smartcontractkit/ccip-owner-contracts/pkg/proposal/timelock"
 
 	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+
 	"github.com/smartcontractkit/chainlink/deployment"
 	"github.com/smartcontractkit/chainlink/deployment/ccip/changeset/internal"
 	commoncs "github.com/smartcontractkit/chainlink/deployment/common/changeset"
 	"github.com/smartcontractkit/chainlink/deployment/common/proposalutils"
+	"github.com/smartcontractkit/chainlink/v2/core/capabilities/ccip/ccipevm"
 	cctypes "github.com/smartcontractkit/chainlink/v2/core/capabilities/ccip/types"
 	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/ccip/generated/fee_quoter"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/ccip/generated/nonce_manager"
 	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/ccip/generated/offramp"
 	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/ccip/generated/onramp"
 	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/ccip/generated/router"
 )
 
 var (
-	_ deployment.ChangeSet[UpdateOnRampDestsConfig]    = UpdateOnRampsDests
-	_ deployment.ChangeSet[UpdateOffRampSourcesConfig] = UpdateOffRampSources
-	_ deployment.ChangeSet[UpdateRouterRampsConfig]    = UpdateRouterRamps
-	_ deployment.ChangeSet[UpdateFeeQuoterDestsConfig] = UpdateFeeQuoterDests
-	_ deployment.ChangeSet[SetOCR3OffRampConfig]       = SetOCR3OffRamp
+	_ deployment.ChangeSet[UpdateOnRampDestsConfig]     = UpdateOnRampsDests
+	_ deployment.ChangeSet[UpdateOffRampSourcesConfig]  = UpdateOffRampSources
+	_ deployment.ChangeSet[UpdateRouterRampsConfig]     = UpdateRouterRamps
+	_ deployment.ChangeSet[UpdateFeeQuoterDestsConfig]  = UpdateFeeQuoterDests
+	_ deployment.ChangeSet[SetOCR3OffRampConfig]        = SetOCR3OffRamp
+	_ deployment.ChangeSet[UpdateFeeQuoterPricesConfig] = UpdateFeeQuoterPricesCS
```

Comment:
Appending CS at the end of changesets. otherwise in lot of places the CS and deploy/config funcs are throwing lint error for having same name